docker/test/integration/minifi/validators/NumFileRangeValidator.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import os
+
+from .FileOutputValidator import FileOutputValidator
+
+
+class NumFileRangeValidator(FileOutputValidator):
+
+    def __init__(self, min_files, max_files):
+        self.valid = False
+        self.min_files = min_files
+        self.max_files = max_files
+
+    def validate(self):
+        self.valid = False
+        full_dir = os.path.join(self.output_dir)
+        logging.info("Output folder: %s", full_dir)
+
+        if not os.path.isdir(full_dir):
+            return self.valid
+
+        num_files = self.get_num_files(full_dir)
+        self.valid = self.min_files < num_files and num_files < self.max_files
+        return self.valid
```

Comment:
I would also get rid of `self.valid` here:
```suggestion
        if not os.path.isdir(full_dir):
            return False

        num_files = self.get_num_files(full_dir)
        return self.min_files < num_files and num_files < self.max_files
```

(and similarly in `SingleFileOutputValidator` and `SingleOrMoreFileOutputValidator`)

Reply:
Update in cdb014f306f614bfad5e6120f089ab821c904677